src/js/background.js

```diff
@@ -776,6 +780,25 @@ function startBackgroundListeners() {
     }
   }, {urls: ["http://*/*", "https://*/*"]}, []);
 
+  // Temporary fix for android while it doesn't support `browser_action.default_popup`
+  // Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1330159
+  if(!chrome.browserAction.getPopup) {
+    chrome.browserAction.onClicked.addListener(() => {
+      chrome.tabs.query({active: true, lastFocusedWindow: true}, (tabs) => {
+        var url = chrome.runtime.getManifest().browser_action.default_popup + "?id=" + tabs[0].id;
+
+        // check if popup already is open
+        chrome.tabs.query({url}, (popupTabs) => {
+          if(popupTabs.length > 0){
+            chrome.tabs.update(popupTabs[0].id, {active: true});
+            chrome.tabs.reload(popupTabs[0].id);
+          } else {
+            chrome.tabs.create({url, index: tabs[0].index + 1});
```

Comment:
Actually, I don't think that would work... Another idea is using `sendMessage`. But if this is working well there is no need to fix it.